src/Components/Button/style.module.css

```diff
@@ -0,0 +1,8 @@
+.button {
+  border: solid 1px #1AAD6F;
+  background: #1AAD6F;
+  color: #FFFFFF;
```

Comment:
Temos um site que mostra os nomes das cores [name-that-color]( https://chir.ag/projects/name-that-color/#AAAAAA)

No projeto temos um arquivo para adicionar as cores que precisamos com a variaveis root do css
[https://github.com/JSL-Future/jsl-2.0/blob/master/src/assets/css/colors.css](https://github.com/JSL-Future/jsl-2.0/blob/master/src/assets/css/colors.css)

Para criar uma variavel root no css adicione o codigo baixo no arquivo de color e adicione as cores que vc precisa, pegando o nome do site que fala o nome da core exemplo abaixo como seria
```css
:root {
  --main-bg-color: brown;
}
```
Para usar o arquivo de cores precisamos colocar importa o css dentro do arquivo css do botão

```css
@import 'caminho do arquivo de cores';
```

O botão não tem o efeito de hover e nem o aparecer o cursor do tipo pointer